src/main/navigation/navigation.c

```diff
@@ -114,6 +114,7 @@ PG_RESET_TEMPLATE(navConfig_t, navConfig,
             .rth_alt_control_override = SETTING_NAV_RTH_ALT_CONTROL_OVERRIDE_DEFAULT, // Override RTH Altitude and Climb First using Pitch and Roll stick
             .nav_overrides_motor_stop = SETTING_NAV_OVERRIDES_MOTOR_STOP_DEFAULT,
             .safehome_usage_mode = SETTING_SAFEHOME_USAGE_MODE_DEFAULT,
+            .mission_planner_resume = SETTING_NAV_MISSION_PLANNER_RESUME_DEFAULT,     // resume WP entry when Mission Planner mode is restarted
```

Comment:
Please update PG version (12 => 13). We all forget this.

Reply:
Done ...